app/views/posts/show.html.erb

```diff
@@ -1 +1,32 @@
-<h2> Posts <%= params[:id] %>for given user </h2>
\ No newline at end of file
+<h2 class='heading'> Posts for user number <%= params[:id] %></h2>
+<div class='post-details-body'>
+    <div class='post-details-p-card'>
+        <div class='post-details-top'>
+            <h3>
+                <%= @post.title %> by <%= @post.user.name %>
+            </h3>
+
+            <div class='post-com-lik-counters'>
+                <p>
+                    Comments: <%= @post.comments_counter%>
+                </p>
+                <p>
+                    Likes: <%= @post.likes_counter%>
+                </p>
+            </div>
+        </div>
+
+        <p class='post-details-text'>
+            <%= @post.text %>
+        </p>
+    </div>
+
+    <div class='post-details-p-comments'>
+        <% @comments.each do |comment| %>
+            <p>
+                <%= comment.user.name%>: <%= comment.text%>
+            </p>
+        <% end %>
+    </div>
```

Comment:
- [ ] Kindly note that `<%= comment.user.name%>` this relationship does not exist in your comment table, if you want to get the user name kindly refer to the `<%= @user.name %>` which is defined in the post#show controller. I am getting this error :point_down: 

![image](https://user-images.githubusercontent.com/65068771/156240358-822541b7-132d-40e0-b2bf-c2a718658aec.png)